relay.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
```

Comment:
Please add option --help (-h) to the script and print out short description with the usage of the command